tests/parametric/test_128_bit_traceids.py

```diff
@@ -427,30 +429,31 @@ def test_w3c_128_bit_propagation_tid_consistent(self, test_agent, test_library):
 
     @missing_feature(context.library == "ruby", reason="not implemented")
     @missing_feature(context.library == "nodejs", reason="not implemented")
-    @missing_feature(context.library == "dotnet", reason="not implemented")
     @missing_feature(context.library == "java", reason="not implemented")
     @pytest.mark.parametrize(
         "library_env",
         [{"DD_TRACE_PROPAGATION_STYLE": "tracecontext", "DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED": "true",}],
     )
-    def test_w3c_128_bit_propagation_tid_in_parent(self, test_agent, test_library):
+    def test_w3c_128_bit_propagation_tid_in_chunk_root(self, test_agent, test_library):
         """Ensure that only root span contains the tid.
         """
         with test_library:
             with test_library.start_span(name="parent", service="service", resource="resource") as parent:
-                with test_library.start_span(name="child", service="service", parent_id=parent.span_id):
+                with test_library.start_span(name="child", service="service", parent_id=parent.span_id) as child:
                     pass
 
-        traces = test_agent.wait_for_num_traces(1, clear=True)
-        parent = find_span_in_traces(traces, Span(name="parent", service="service"))
-        child = find_span_in_traces(traces, Span(name="child", service="service"))
+        traces = test_agent.wait_for_num_traces(1, clear=True, sort_by_start=False)
+        trace = find_trace(traces, parent.trace_id)
+        assert len(trace) == 2
+        chunk_root = find_chunk_root_span(trace)
+        non_chunk_root = trace[1]
 
-        parent_tid = parent["meta"].get("_dd.p.tid")
-        child_tid = (child.get("meta") or {}).get("_dd.p.tid")
-        propagation_error = parent["meta"].get("_dd.propagation_error")
+        tid_chunk_root = chunk_root["meta"].get("_dd.p.tid")
+        tid_non_chunk_root = (non_chunk_root.get("meta") or {}).get("_dd.p.tid")
```

Comment:
Just curious (not a Python expert), why access `meta` in different ways here?
```
chunk_root["meta"]
non_chunk_root.get("meta")
```

Reply:
chunk_root["meta"] will raise a KeyError Exception if meta is not found in map 

non_chunk_root.get("meta") will return None if meta is not found in the map.

We are pretty inconsistent in system tests. We should stick to one convention. In my opinion we use .get for safe access